src/adagrams.js

```diff
@@ -1,8 +1,99 @@
 const Adagrams = {
+  letters : "abcdefghijklmnopqrstuvwxyz".toUpperCase().split(""),
+  letterQuantity : [9, 2, 2, 3, 12, 2, 3, 2, 9, 1, 1, 4, 2, 6, 8, 2, 1, 6, 4, 6, 4, 2, 2, 1, 2, 1],
+  letterValues: {"A": 1, "E": 1, "I" : 1, "O" : 1, "U": 1, "L": 1, "N": 1, "R" : 1, "S" : 1, "T" : 1, "D" : 2, "G" : 2, "B" : 3, "C" : 3, "M" : 3, "P" : 3, "F" : 4, "H" : 4, "V" : 4, "W" : 4, "Y" : 4, "K" : 5, "J" : 8, "X" : 8, "Q" : 10, "Z" : 10},
```

Comment:
Instead of storing the letters and the letter quantities separately, could you store them as a hash:

```
letterQuantities: {
  A: 9, B: 2, ...
}
```

That would make this code easer to change, since you wouldn't need to mess with those parallel arrays. For example, what if you noticed you had a bug and the number of 'u' characters was wrong - which way would make the problem easier to fix?